router/throttler/switcher.go

```diff
@@ -0,0 +1,37 @@
+package throttler
```

Comment:
@atzoum Is it required to hot reload between the two implemantioans?

When we the switch happens, the previous throttles will keep running in the background:
a. the use of memory and CPU resources
b. they stop getting updates on what is going on, which might lead to unknown/unpredictable behaviour if we switch back to them.

From my perspective, the probability of switching between those implementations will be low and restarting the processer is not a big issue. Given the additional implementation and risk for switching between implementations. 

I would suggest removing this package, and resorting to restarts if needed.